examples/custom-client-cxx/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required (VERSION 3.5)
+project(custom-sota-client)
+
+add_executable(custom-sota-client main.cc)
```

Comment:
I just wonder why it cannot be another target (custom-sota-client) in the scope of the existing project (aklite).